pkg/core/resources/model/utils.go

```diff
@@ -0,0 +1,94 @@
+package model
+
+import (
+	"encoding/json"
+	"path"
+	"reflect"
+
+	"github.com/ghodss/yaml"
```

Comment:
Isn't yaml2 the right yaml to use (I always get confused :) )

Reply:
We have 60 matches for `"github.com/ghodss/yaml"` and 5 matches for `"gopkg.in/yaml.v2"`, I don't know if there are any limitations why we can't stick to 1 library. I can create an issue for that

Reply:
https://github.com/kumahq/kuma/issues/5192